docs/manual/en/introduction/Useful-links.html

```diff
@@ -55,7 +55,7 @@ <h3>More extensive / advanced articles and courses</h3>
 				[link:https://medium.com/soffritti.pierfrancesco/glossy-spheres-in-three-js-bfd2785d4857 Glossy spheres in three.js].
 			</li>
 		 <li>
-			 [link:https://www.udacity.com/course/cs291 Interactive 3D Graphics] - a free course on Udacity that teaches the fundamentals of 3D Graphics,
+			 [link:https://www.udacity.com/course/interactive-3d-graphics--cs291 Interactive 3D Graphics] - a free course on Udacity that teaches the fundamentals of 3D Graphics,
```

Comment:
Do you mind updating the broken link in the translated pages as well?

Reply:
done.